markdownlint/TOP002_noCodeInHeadings/TOP002_noCodeInHeadings.js

```diff
@@ -13,9 +13,17 @@ module.exports = {
     );
 
     headingsWithCode.forEach((heading) => {
+      const codeContent = heading.line.match(/`.+?`/);
```

Comment:
Could we add a comment with a regexr link with tests, as per the other rules with regex?
Could also probably do `` /`.*`/ `` to the same effect, no?